Kitodo/src/main/java/org/kitodo/production/interfaces/activemq/ActiveMQDirector.java

```diff
@@ -65,21 +64,6 @@ public class ActiveMQDirector implements Runnable, ServletContextListener {
     private static Session session = null;
     private static MessageProducer resultsTopic;
 
-    /**
-     * The method is called by the web container on startup
-     * and is used to start up the active MQ connection. All processors from
-     * {@link #services} are registered.
-     */
-    @Override
-    public void contextInitialized(ServletContextEvent initialisation) {
```

Comment:
Not sure how Checkstyle did let this pass, but removing this code renders the corresponding import statements for `ServletContextEvent`, `ServletContextListener` and `WebListener` obsolete. Therefore, these now-unused import statements should be removed.

Reply:
Yes, that's right.